update-aiida-diff.sh

```diff
@@ -11,8 +11,6 @@ else
 fi
 
 cookiecutter --no-input -f . version=1.2.0
-pip install -e aiida-diff[docs,pre-commit,testing]
 cd aiida-diff
 git init && git add -A
-pre-commit install
-pre-commit run
+hatch fmt
```

Comment:
Old: runs linters, New: doesn't
```suggestion
hatch fmt --check
```

Reply:
Seems fine. Also, my original comment was wrong anyway.